### ISSUE TITLE
Improve deliverability of email

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#674]
# TL;DR
Emails, including compensation emails, are often getting filtered as spam. 

# Narrative
As a researcher, I want to be confident I'm actually reaching participants when I use the Lookit interface to send messages. This affects our reputation as well. As a participant, I want to receive email from researchers, particularly when that email contains compensation for a study we participated in.

# To consider:

- [ ] Figure out how often this is happening (may need to talk with researchers who get direct complaints)
- [ ] Figure out why emails are getting flagged as spam (is it our language? headers? from address vs. reply_to?)
- [ ] Can we use an MIT SMTP server instead?
- [ ] Would authenticating a domain via SendGrid help if sticking with SendGrid?

**Comment by becky-gilbert** [comment]
Closing this issue as it is superseded by https://github.com/lookit/lookit-api/issues/1340
